### ISSUE TITLE
64 bit requirement play store

### ISSUE THREAD

**Issue by Gonsaave** [niqdev/ipcam-view#91]
Hi niqdev, hope you're doing well in these times.

I'm trying to publish an app using your library, but I can't due to the new 64 bit requeriment. I noticed there was another issue related to this, it was solved with a PR but it doesn't work for me.

I'm using version 1.7.0 of the library, maybe i'm missing something else?

Gradle:

implementation 'com.github.niqdev:mjpeg-view:1.7.0'

Any help is appreciated, thanks and stay safe.

**Comment by niqdev** [comment]
Hi @Gonsaave,
thanks, I hope you are doing well too!

Honestly I'm not actively developing with android in these days, although I have 2 apps on the store that use this lib, so it's in my interest to maintain at least compatibility and the project alive.

I don't know much about the new 64 bit requirements, but if you are able to figure it out and open a pr, I would be more than happy to merge any pr and release a new version of the lib. How does that sounds? I can help to point out to specific part of the code if you need it, also keep in mind that the NDK is not required to build the project and not used anymore.

Please let me know if you need any details to narrow down the issue

**Comment by Gonsaave** [comment]
First off, thanks for the quick response! 

I recently got back to Android development and saw all the changes in google politics, android 10 and other things, quite a lot has changed honestly. Although i'm no android expert, but i'll glady help if I can, so i'll check this and see if i can find anything (maybe it's something just related to my app even).

Again, any help is appreciated, i'll let you know if i need help, thanks!

**Comment by Gonsaave** [comment]
Sorry for taking so long, was busy during the week. So i was able to give it a look during the weekend and after some research and testing, apparently the thing needed was to recompile the NDK with x86_64 added.

Correct me if i'm wrong @niqdev, i just modified the file Application.mk for mjpeg-view, recompiled and checked the APK with APK Analyzer (Android Studio) and everything seems alright now. Am i right or is there something else i'm missing?

**Comment by niqdev** [comment]
That's great news!
I would expect that you have done something similar to #81, isn't it? If that's the case and it works, would you mind opening a pr for that? I can publish a new version of the lib so that you can give me a feedback, wdyt?

**Comment by Gonsaave** [comment]
Yes it's very similar to #81 (added x86_64 so the NDK compiles for 32 and 64 bits now). I'll open a new PR and i'll give you feedback on how it goes!

**Comment by jpage4500** [comment]
Hi @Gonsaave - any updates on this? i was looking for a good mjpg view to use for my app and this one seems very robust - but I didn't want to add it and then realize I can't use it in Google Play

**Comment by casic** [comment]
Hi @Gonsaave  - is there  already any solution ???
Best regards ?

**Comment by casic** [comment]
For all with this  problem 
Just use  Mac or Linux .   Open  Application.mk   ,  add  x86_64  .  Run  ./compileJni.sh  
New folder  will  appear in JniLbs /x86_64.
I hope this help someone

**Comment by Gonsaave** [comment]
Hi everyone, i'm really sorry for leaving this issue pending all this time. Last year got really rough in my workplace and I couldn't come back to this. If anyone is still interested, I double checked my fix and it should work.

I can make a PR for this.

Again, i'm really sorry to everyone. Best regards.
